### DISCUSSION TITLE
Jira deployments

### DISCUSSION THREAD

**Discussion by ari** [buildbot/buildbot#6297] [Ideas]
Has anyone built an integration for Jira deployments?

https://www.atlassian.com/software/jira/guides/developers/ci-cd#connect-your-cicd-tools

There is [an API](https://www.atlassian.com/software/jira/guides/developers/ci-cd#connect-your-cicd-tools) which looks simple enough but I thought I'd ask in case someone has had a go at this integration.

**Comment by p12tic** [comment]
I'm not aware of anything yet. It should be relatively easy to write a reporter for Jira.

**Comment by ari** [comment]
Actually it looks like it might not be possible. The API looks like it is only available to certain reigstered apps: https://developer.atlassian.com/cloud/jira/software/rest/api-group-deployments/

Perhaps that would be available for buildbot as a project to integrate with, but not sure we can create a custom connector.
